web/src/components/shared/Item/Item.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './item.css';
+// import { Link } from 'react-router-dom';
+
+export type ItemType = {
+    title: string;
+    imageUrl?: string;
+    price?: number;
+    id?: string;
```

Comment:
should be
url?: string

also we need decription (see example image in the ticket)